plugins/qcheck-stm/src/stm_of_ir.ml

```diff
@@ -120,6 +120,65 @@ let mk_cmd_pattern value =
   let name = String.capitalize_ascii (str_of_ident value.id) |> lident in
   ppat_construct name args
 
+let ty_show_of_core_type inst ty =
```

Comment:
This function could be structured (and named) closer to the equivalent one for `next_state`, `mk_pat_ret_ty`. In particular it would be nice to put them close together and to reuse `munge_longident` (pulling the `capitalize_ascii` out as an argument, I suppose), as they should do the same thing apart from the pat/exp target.